packages/webcert/src/feature/certificate/Buttons/Test/RevokeCertificateButton.test.tsx

```diff
@@ -180,4 +198,47 @@ describe('Revoke continue button', () => {
       type: '[CERTIFICATE] Revoke certificate',
     })
   })
+
+  describe('isDodsbevis', () => {
+    const mockDispatchFn = jest.fn()
```

Comment:
Can use dispatchHelperMiddleware instead of using mocks. The dispatchHelperMiddleware just stores the different actions that has been dispatched and exposes them for you to assert/expect what you need. 

Here is just an example from another test:
```
    it('should throw hide action when hidden', async () => {
        const certificate = showValidation(false, true)
        testStore.dispatch(updateCertificate(certificate))

        testStore.dispatch(validateCertificateInFrontEnd(certificate.data['1.1']))

        const hideCertificateDataElementAction = dispatchedActions.find((action) => hideCertificateDataElement.match(action))
        expect(hideCertificateDataElementAction && hideCertificateDataElementAction.payload).toBe('1.2')
      })
```